lib_examples/base64_example/README.md

```diff
@@ -0,0 +1,28 @@
+# Base64 example use and bb benchmarking
+
+This makes use of [this](https://github.com/noir-lang/noir_base64.git) base64 library (see Nargo.toml for version)
+
+The lib covers several encode/decode tests, whereas this program has some extra code to facilitate benchmarking the gate counts of a proving backend. Comparisons can be made against other proving backends if desired.
+
+Ordinarily base64 is used encode/decode binary data sent across the web (in html/css files), rather than the text strings shown here. So could be used to verify that a website sent a particular image by comparing to a local binary, with applications that could equate to digital watermarks.
```

Comment:
Interesting idea. just curious if you have ideas about when someone might want to do this in practice?

Reply:
Referred to email/browser use